pom.xml

```diff
@@ -103,6 +108,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.organization.management.core</groupId>
+                <artifactId>org.wso2.carbon.identity.organization.management.service</artifactId>
+                <version>${org.wso2.carbon.identity.organization.management.core.version}</version>
+            </dependency>
```

Comment:
Where is this dependency used?

Reply:
This dependency was introduces because the AuthenticatedUser class has imported some classes from this dependency and unit tests are getting failed after bumping the framework version to the latest version.